src/inpar/4C_inpar_structure.hpp

```diff
@@ -713,7 +713,7 @@ namespace Inpar
         case Inpar::Solid::KinemType::linear:
           return "linear";
         case Inpar::Solid::KinemType::nonlinearTotLag:
-          return "nonlinear_total_lagrange";
+          return "nonlinear";
       }
 
```

Comment:
We also have Updated Lagragian formulation. Do you think it will be extended to those. If that is the case, `total_lagrange` would be a better name?